nixos/modules/config/pulseaudio.nix

```diff
@@ -210,6 +218,11 @@ in {
     }
 
     (mkIf cfg.enable {
+      assertions = [{
+        assertion = !cfg.support32Bit;
+        message = "`hardware.pulseaudio.support32Bit` is no longer supported.";
```

Comment:
> The alsa 32bit options have been pretty widely recommended in the past due to steam so I'd like to see a proper deprecation error instead of just silently removing the options.

@jansol That makes sense. This commit adds a clearer message, but did you mean an actual deprecation process, where we fix the currently broken 32bit support and provide a deprecation warning about its eventual removal? That would require reintroducing the patch that was removed in #154276, but I'm not opposed to doing that if it's warranted (and it sounds like it is).

I'd also like to provide a more informative message than I've done in this commit, something like,
```
In order to support 32bit audio on a 64bit system, please use ??? instead.
```
But I don't know enough about this domain to do so. Is there a pipewire-related suggestion that would be appropriate in place of `???` above?
cc @L-as